src/modules/interact.js

```diff
@@ -0,0 +1,22 @@
+export const baseURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/';
+export const appId = 'byg3KtvqOhmd3Xt9Axu5';
+
+export const addLike = async (itemId) => {
+  fetch(`${baseURL}apps/${appId}/likes`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ item_id: itemId }),
+  });
+};
```

Comment:
- [ ] The counters functions should get covered with unit tests using [the Jest Framwework](https://jestjs.io/). Kindly add this, Here is the error message I got after I runing the tests in my computer 👇🏻 Kindly address this.

![scree24](https://user-images.githubusercontent.com/79058364/180600271-7568d6ca-dd00-4e31-af5f-59a2665c14d2.png)